<a href="https://colab.research.google.com/github/goulartsursula/python-uerj/blob/master/ex003.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Exercício 003:** Se fizermos uma corrida de 10 quilômetros em 43 minutos e 30 segundos, qual será o seu tempo médio por milha? Qual é a sua velocidade média em milhas por hora?

In [1]:
43 * 60

2580

10 quilômetros em 2610 segundos

In [4]:
(1.61 * 2610) / 10

420.21000000000004

Tempo médio por milha é de 420.21 segundos, aproximadamente 7 minutos.

In [9]:
10 / 1.61

6.211180124223602

10km = 6.21 mi

In [10]:
(2610 / 3600)

0.725

43 minutos e 30 segundos é equivalente a 0.725 horas.



In [11]:
6.21 / 0.725

8.565517241379311

Velocidade média de 8.566 mi/h